README.md

```diff
@@ -9,22 +11,38 @@ Issues and feature requests posted to GitHub are not stored in any
 publicly available location, so I find that I quickly lose track of
 the things that I've sent them.
 
+## For GitHubbers
+
 If you are a GitHubber, and you would like to respond and close issues
 on this repository, please let me (or one of the other collaborators)
 know, and we'll add you as a collaborator.  Or maybe use some secret
```

Comment:
you could change this to something along the lines of 
```
know, and we'll add you as a collaborator (or maybe use some secret...
```
Either way, there's a double space between the period and `Or`. I think parentheses would be a better fit here.

Reply:
Double space is fixed.

There is another parentheses before that, so I think I having another one looks weird.

Personally, I would drop this sentence entirely:

> Or maybe use some secret GitHub internal APIs to get access, if you have such things.

But I think that is beyond the scope of this PR which was more about restructuring.

Reply:
In English (i.e., not various programming languages 😉),  if nested parentheses are _needed_ (usually such can be reworded to avoid), the outermost can often be replaced by long dash ("—", which has some specific technical name I forget 🤔), &/or the inner most can be done in order of "([{}])."  Maybe try such here?

Reply:
I did not mean nested parentheses. I meant 2 pairs of parentheses in rapid succession:

---

Original text:

> please let me _(or one of the other collaborators)_ know, and we'll add you as a collaborator.  Or maybe use some secret

With proposed change:

> please let me _(or one of the other collaborators)_ know, and we'll add you as a collaborator _(or maybe use some secret_ ...

---

Using parenthesese or the long dash (I don't know the correct name either) - either way it is adding some subsentence which - especially if overused - does not improve readability (which should in most case be tried to achieve so that people _read the text_) ... doing this now to make my point :wink:

If you ask me, I would heavily shorten the text in general, I would change `me (or one of the other collaborators)` into `us` etc. but that would be a lot for one change. So I would prefer to create another PR for that (if at all). And I have the very strong feeling @safinsingh is better at this than I am.

So, I advise against it, but will oblige if you insist. In any cases I would be delighted if someone else wishes to make the additional changes and finish this up.